docs/docs/recipes.md

```diff
@@ -29,6 +29,12 @@ A live example may not be possible depending on the nature of the recipe, in whi
 See [docs templates](/docs/docs-templates/) in the contributing docs for more help.
 -->
 
+## New automated recipes available!
+
+Some recipes are now available in a more automated form - you can run some steps of the recipes through the CLI! [See a list of these automated recipes here](/docs/automated-recipes.md).
```

Comment:
I was thinking something like this? Probably needs some wordsmithing, but since we don't have full coverage on the recipes listed here I'm not sure we're ready to conflate the two.


You may have found this page when searching for the new automated Gatsby recipes! Note that these are still experimental and thus can be found in the [Gatsby GitHub repo recipes package](https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby-recipes/README.md).

Reply:
oh nice I like that!